### ISSUE TITLE
Make bot features optional with environment variables

### ISSUE THREAD

**Issue by loriopatrick** [playit-gg/discord-feature-bot#6]
For the playit.gg discord, I don't want to have the anti spam component running util we think it's needed. It would be nice to have the different components of the bot feature gated by environment variables.

**Comment by Alienwarez123** [comment]
Ill work on it

**Comment by Alienwarez123** [comment]
done
